site/content/docs/01-component-format.md

```diff
@@ -277,6 +277,12 @@ To apply styles to a selector globally, use the `:global(...)` modifier.
 			 to this component */
 		color: goldenrod;
 	}
+
+	p:global(.red) {
+		/* this will apply to <p> elements belonging to this 
+			 component that have the class "red" dynamically
+			 applied (e.g. via node.classList.add) */
```

Comment:
Suggestion: I'd change this to say that is applies to all p elements in this component with class red and explain in a second sentence that "this is useful when .."

Reply:
Thanks for the feedback. I've updated the wording per your suggestion.